.kitchen.yml

```diff
@@ -6,15 +6,18 @@ provisioner:
   deprecations_as_errors: true
   # Uncomment when testing idempotency
   # The add & delete key in dummy_key.rb breaks this right now (on purpose)
-  # enforce_idempotency: true
-  # multiple_converge: 2
+  enforce_idempotency: true
+  multiple_converge: 2
 
 verifier: inspec
 
 platforms:
   - name: amazon-linux
     driver_config:
       box: mvbcoding/awslinux
+  - name: amazon-linux-2
+    drive_config:
+      box: winky/amazonlinux-2
```

Comment:
Does this one work out of the box. So many of these are busted or require the proprietary USB 2 module in vbox.